### DISCUSSION TITLE
Inappropriate submissions

### DISCUSSION THREAD

**Discussion by MrKraken** [pwillworth/galaxyharvester#47] [General]
@pwillworth I've just renamed and removed 2 inappropriate  resource submissions, however there's also a username submitting them that is directly referencing a player/streamer in the community. Have emailed the GH gmail with details.

Tagged here again just incase you are still having notification issues for that mail

**Comment by pwillworth** [comment]
Thanks for the heads up and cleaning those up, @MrKraken .  I'm doing what I can on the server side.
